Source/Scene/Scene.js

```diff
@@ -1310,6 +1456,15 @@ define([
 
             scene._globeDepth.executeCopyDepth(context, passState);
 
+            if (scene.debugShowGlobeDepth) {
```

Comment:
>  I don't see how you avoid the copy without just using _globeDepth

The new `_debugGlobeDepths[]` would have the globe depth (basically `_globeDepth`) for each frustum so the inspector can index into it and show the right one.  Why is a copy needed?  Isn't `_globeDepth` currently just depth for the globe (and overwritten per frustum)?  It is already copied above:
```
scene._globeDepth.executeCopyDepth(context, passState);
```